templates/tower-project.yaml

```diff
@@ -108,6 +108,69 @@ Resources:
             Principal:
               AWS: !Ref AccountAdminArns
 
+  TowerForgeBatchHeadJobRole:
+    Type: AWS::IAM::Role
+    Properties:
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/AmazonS3ReadOnlyAccess
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Action: sts:AssumeRole
+            Principal:
+              Service:
+                - ecs-tasks.amazonaws.com
+
+  TowerForgeBatchWorkJobRole:
+    Type: AWS::IAM::Role
+    Properties:
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/AmazonS3ReadOnlyAccess
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Action: sts:AssumeRole
+            Principal:
+              Service:
+                - ecs-tasks.amazonaws.com
+
+  TowerForgeBatchHeadJobPolicy:
+    Type: AWS::IAM::Policy
+    Properties:
+      PolicyName: TowerForgeBucketAccessPolicy
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: NextflowConfig
+            Effect: Allow
+            Action:
+              - batch:DescribeJobQueues
+              - batch:CancelJob
+              - batch:SubmitJob
+              - batch:ListJobs
+              - batch:DescribeComputeEnvironments
+              - batch:TerminateJob
+              - batch:DescribeJobs
+              - batch:RegisterJobDefinition
+              - batch:DescribeJobDefinitions
+              - ecs:DescribeContainerInstances
+              - ecs:DescribeTasks
+              - ec2:DescribeInstances
+              - ec2:DescribeInstanceAttribute
+              - ec2:DescribeInstanceTypes
+              - ec2:DescribeInstanceStatus
+            Resource: "*"
```

Comment:
Going to approve this PR, but it's too bad this has to be a star. Ideally the batch permissions should only apply to the batch created for this Tower compute environment. But this is created before that exists. This would be one advantage of manual setup over forge.